grafana-plugin/package.json

```diff
@@ -3,8 +3,8 @@
   "version": "1.0.0",
   "description": "Grafana OnCall Plugin",
   "scripts": {
-    "lint": "eslint --cache --ext .js,.jsx,.ts,.tsx --max-warnings=0 ./src ./e2e-tests",
-    "lint:fix": "eslint --max-warnings=0 --fix --cache --ext .js,.jsx,.ts,.tsx ./src ./e2e-tests",
```

Comment:
eslint is quite fast anyway and I encountered cases when updates in node_modules affect linting. At least for now I would feel more comfortable making sure we always lint the whole codebase